addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -465,7 +466,13 @@ private void updateMediaStatus(final MediaStatus mediaStatus) {
         updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_CURRENT_TIME),
                 new DecimalType(mediaStatus.currentTime));
 
-        updateMediaInfoStatus(mediaStatus.media);
+        // If we're playing or paused but don't have any MEDIA information don't null everything out.
+        Media media = mediaStatus.media;
+        if (media == null && (mediaStatus.playerState == PlayerState.PLAYING || playerState == PlayerState.PAUSED)) {
```

Comment:
Why `mediaStatus.playerState` vs `playerState` are the refering to the same playerState somehow or is this an error?